Roslyn.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28620.130
+# Visual Studio 15
```

Comment:
Is this intentional?

Reply:
Yes / No: the version changes whenever you open the SLN file in a new version of Visual Studio and save it. Generally we don't pay much attention to it as fixing it just means it will change for the next person. I'll revert this though.